tally_ho/apps/tally/templates/reports/supervisor_approvals.html

```diff
@@ -13,7 +13,7 @@ <h3>{% trans "Audit Supervisors Approval Rate" %}</h3>
         <th>{% trans 'Number of Forms Sent For Review' %}</th>
         <th>{% trans 'Approval Rate' %}</th>
     </tr>
-    {% if a_s_approvals.forms_approved == 0 %}
+    {% if not a_s_approvals.forms_approved %}
     <tr>
         <td>No Data</td>
```

Comment:
Do we want to make this string translatable?

Reply:
We do

Reply:
Done